src/ripple/app/consensus/RCLCxLedger.h

```diff
@@ -37,6 +37,8 @@ class RCLCxLedger
 public:
     //! Unique identifier of a ledger
     using ID = LedgerHash;
+    //! Sequence number of a ledger
+    using Seq = LedgerIndex;
```

Comment:
Potentially. `LedgerMaster` and `NetworkOps` still have functionality that is related to consensus that should arguably be moved to the generic implementation and that might warrant the change.